ui/src/app/components/buttons.tsx

```diff
@@ -300,7 +300,7 @@ export const StyledAnchorTag = ({href, style = {}, children, ...props}) => {
   return <a href={href}
             onClick={e => {
               // This does same page navigation iff there is no key pressed and target is not set.
-              if (props.target === undefined) {
+              if (props.target === undefined && !href.startsWith('https://') && !href.startsWith('http://')) {
```

Comment:
It would be great if we had another option here, but CH and I couldn't come up with one.